src/components/Accordion/styled.ts

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: MPL-2.0 */
+/* Copyright (C) 2023 Gemeente Amsterdam */
+import { Button as AscButton, themeSpacing } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+export interface Props {
+  isOpen?: boolean
+}
+
+export const Wrapper = styled.div<Props>`
+  border: ${({ isOpen }) =>
+    isOpen ? '1px solid #102e62' : '1px solid transparent'};
+  margin-bottom: 4px;
+
+  &:hover {
+    border: 1px solid #102e62;
+  }
+`
+
+export const Border = styled.div<Props>`
+  border: 1px solid transparent;
+
+  &:hover {
+    border: ${({ isOpen }) =>
+      isOpen ? '1px solid #102e62' : '1px solid #102e62'};
+  }
+`
+
+export const Button = styled(AscButton)<Props>`
+  width: 100%;
+  box-sizing: border-box;
+  svg {
+    transform: rotate(${({ isOpen }) => (isOpen ? '180deg' : '0deg')});
+    transition: transform 0.3s ease;
+  }
+
+  &:hover {
+    color: #102e62;
```

Comment:
might wanne consider a color constant file, that's used for the whole app, good idea? 2024/2025

Reply:
Forgot about that yes. I've put a const in this file now. Let's discuss a way how we want to deal with this since we are deviating form StyledComponents themeColor with the new design system.